src/Button/Button.scss

```diff
@@ -0,0 +1,4 @@
+.Button {
```

Comment:
was just about to comment on that.  Good call, I'm good with this for initial testing.  Nice work.